src/overlays/actors/ovl_Dm_Char09/z_dm_char09.c

```diff
@@ -32,28 +30,197 @@ const ActorInit Dm_Char09_InitVars = {
     (ActorFunc)DmChar09_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_0600005C;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB1E10.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/DmChar09_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/DmChar09_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB1FA0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB1FDC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB2258.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB2268.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB24BC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/DmChar09_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Char09/func_80AB25D8.s")
+static AnimationInfo sAnimations[] = {
```

Comment:
```suggestion
static AnimationInfo sAnimationInfo[] = {
```
Not my decision but we want to change this array name to include `Info`
reference: https://github.com/zeldaret/mm/pull/998